src/assets/CurrencyRateController.ts

```diff
@@ -195,16 +195,27 @@ export class CurrencyRateController extends BaseController<
       pendingCurrentCurrency,
       pendingNativeCurrency,
     } = this.state;
+
+    let conversionDate: number = Date.now() / 1000;
+    let conversionRate: number | null = null;
+    let usdConversionRate: number | null = null;
     try {
-      const {
+      ({
         conversionDate,
         conversionRate,
         usdConversionRate,
       } = await this.fetchExchangeRate(
         pendingCurrentCurrency || currentCurrency,
         pendingNativeCurrency || nativeCurrency,
         this.includeUsdRate,
-      );
+      ));
+    } catch (error) {
+      if (
+        !error?.message.includes('market does not exist for this coin pair')
```

Comment:
The `?` here is what's causing the test coverage to drop. There aren't any test cases where the error itself is nullish.

Perhaps it could just be removed? The error should never be nullish.

```suggestion
        !error.message.includes('market does not exist for this coin pair')
```

Reply:
I tried this locally and it did get tests to pass, but `prettier` demanded that this be moved up a line, so this suggestion won't pass CI as-is.